easy_phi/tests/run_tests.py

```diff
@@ -1,12 +1,46 @@
 # -*- coding: utf-8 -*-
-
+import tornado.httpserver
+import tornado.httpclient
+import tornado.ioloop
+import tornado.web
 import unittest
+import json
+from easy_phi import app as handlers
 
 class API_test(unittest.TestCase):
 
     def test_t(self):
```

Comment:
I will need both especially if you say that I should deal with json instead of the strings.